include/seqan/align/fragment.h

```diff
@@ -65,7 +65,7 @@ struct ExactFragment;
  * @headerfile <seqan/align.h>
  * @brief A type for ungapped, pairwise segment matches that maybe in reverse orientation.
  *
- * Compared to the @link ExactFragment @endlink specialzing type of @link Fragment @endlink, a @link
+ * Compared to the @link ExactFragment @endlink specialized type of @link Fragment @endlink, a @link
```

Comment:
Compared to the @link ExactFragment @endlink type specializing @link Fragment @endlink, a ...